### PR TITLE
Added info about the vertical lines to time series panels 

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

File: docs/dashboards/panels/index.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ Available chart types:
 * [Box Plot](/docs/dashboards/panels/box-plot-charts)
 * [Heat Map](/docs/metrics/metrics-queries/heat-map)
 
+When you move the cursor over the Line, Column, Bar, Area, and Boxplots time series panels, you can see a vertical line across the panel at the same time point.
```
Comment (kimsauce) [inline docs/dashboards/panels/index.md:45]:
```suggestion
When you move the cursor over the Line, Column, Bar, Area, and Box Plot time series panels, you can see a vertical line across the panel at the same time point.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
small change

**Review by kimsauce — APPROVED** [review]

File: docs/dashboards/panels/index.md [hunk 2]
```diff
@@ -42,14 +42,20 @@ Available chart types:
 * [Box Plot](/docs/dashboards/panels/box-plot-charts)
 * [Heat Map](/docs/metrics/metrics-queries/heat-map)
 
+When you move the cursor over the Line, Column, Bar, Area, and Box Plot time series panels, you can see a vertical crosshair line across the panel which displays the time-series data. If other time series panels are also configured with same time range, then when you hover over one time panel, vertical crosshair will be displayed on the other panels as well. A tooltip is used to display the time-series data along with the vertical crosshair line, which also displays information about the combination chart.
+
+:::note
+Vertical crosshair timestamp is not availble for Heatmap and Table panels.
```
Comment (jpipkin1) [inline docs/dashboards/panels/index.md:48]:
```suggestion
Vertical crosshair timestamp is not available for Heatmap and Table panels.
```

File: docs/dashboards/panels/index.md [hunk 3]
```diff
@@ -42,14 +42,24 @@ Available chart types:
 * [Box Plot](/docs/dashboards/panels/box-plot-charts)
 * [Heat Map](/docs/metrics/metrics-queries/heat-map)
 
+When you move the cursor over the Line, Column, Bar, Area, and Box Plot time series panels, you can see a crosshair line across the panel which displays the time-series data. If other time series panels are also configured with same time range, then when you hover over one time panel, crosshair timestamp will be displayed on the other panels as well. A tooltip is used to display the time-series data along with the crosshair, which also displays information about the combination chart.
```
Comment (oshoghi) [inline docs/dashboards/panels/index.md:45]: It's not just same time range but rather overlapping time range. If a panel is last hour and another is last 15 mins, when mousing over the period of time that is common to both, you will see the sync crosshair.

File: docs/dashboards/panels/index.md [hunk 4]
```diff
@@ -42,14 +42,24 @@ Available chart types:
...
+
+:::note
+Crosshair timestamps are not available for Heatmap and Table panels.
```
Comment (oshoghi) [inline docs/dashboards/panels/index.md:48]: They're also not available for any categorical charts, or honeycomb. Not sure if you already mentioned that elsewhere.

**Review by oshoghi — APPROVED** [review]

File: docs/dashboards/panels/index.md (hunk 4 above)
Comment (JV0812) [reply docs/dashboards/panels/index.md:48]: No Okhtay, I missed to mention it. Now I have added these to the same note.

**Pull request merged**